### PR TITLE
Reduce variable scopes

### PR DESCRIPTION
**Pull request by practicalswift**
Reduce variable scopes.

### CODE REVIEW

File: src/qt/recentrequeststablemodel.cpp [hunk 1]
```diff
@@ -139,10 +139,9 @@ bool RecentRequestsTableModel::removeRows(int row, int count, const QModelIndex
 
     if(count > 0 && row >= 0 && (row+count) <= list.size())
     {
-        const RecentRequestEntry *rec;
         for (int i = 0; i < count; ++i)
         {
-            rec = &list[row+i];
+            const RecentRequestEntry *rec = &list[row+i];
```
Comment (promag) [inline src/qt/recentrequeststablemodel.cpp]: `RecentRequestEntry* rec`

**Review by promag — COMMENTED** [review]
utACK.

File: src/qt/recentrequeststablemodel.cpp [hunk 2]
```diff
@@ -139,10 +139,9 @@ bool RecentRequestsTableModel::removeRows(int row, int count, const QModelIndex
 
     if(count > 0 && row >= 0 && (row+count) <= list.size())
     {
-        const RecentRequestEntry *rec;
         for (int i = 0; i < count; ++i)
         {
-            rec = &list[row+i];
+            RecentRequestEntry *rec = &list[row+i];
```
Comment (Empact) [inline src/qt/recentrequeststablemodel.cpp]: const?

**Review by dcousens — APPROVED** [review]

**Review by randolf — APPROVED** [review]

File: src/qt/recentrequeststablemodel.cpp (hunk 2 above)
Comment (practicalswift) [reply src/qt/recentrequeststablemodel.cpp]: cc @promag :-)

File: src/qt/recentrequeststablemodel.cpp (hunk 1 above)
Comment (practicalswift) [reply src/qt/recentrequeststablemodel.cpp]: cc @Empact :-)

Comment (promag) [reply src/qt/recentrequeststablemodel.cpp]: Ah I didn't mean to remove const, sorry, just the space..

**Comment by practicalswift** [comment]
Please re-review :-)

**Review by randolf — APPROVED** [review]

**Comment by Empact** [comment]
utACK 6a318e4

**Comment by MarcoFalke** [comment]
utACK 6a318e48a67a8f9834e14c960bd4f5b6e295b2dc

**Pull request merged**